### ISSUE TITLE
Simple module tutorial fails

### ISSUE THREAD

**Issue by gdhgdhgdh** [hashicorp/terraform#706]
Hi, I'm following the simple module tutorial at `http://www.terraform.io/docs/modules/create.html` and finding that it doesn't work as described (using current dev compile 0.3.6-dev)

```
root@eddie:~/terraform/r# ls -al
total 16
drwxr-xr-x 3 root root 4096 Dec 23 14:30 .
drwxr-xr-x 5 root root 4096 Dec 23 14:30 ..
drwxr-xr-x 2 root root 4096 Dec 23 14:31 child
-rw-r--r-- 1 root root  127 Dec 23 14:33 main.tf
root@eddie:~/terraform/r# ls -al child/
total 12
drwxr-xr-x 2 root root 4096 Dec 23 14:31 .
drwxr-xr-x 3 root root 4096 Dec 23 14:30 ..
-rw-r--r-- 1 root root   72 Dec 23 14:31 main.tf

root@eddie:~/terraform/r# cat main.tf
module "child" {
    source = "./child"

    memory = "1G"
}

output "child_memory" {
    value = "${module.child.received}"
}

root@eddie:~/terraform/r# cat child/main.tf
variable "memory" {}

output "received" {
    value = "${var.memory}"
}
root@eddie:~/terraform/r# terraform apply
Error downloading modules: module child: not found, may need to be downloaded
```

A quick look at `strace -f terraform apply` output *did* show lstat() for the `child` directory 

```
[pid 20313] lstat("/root/terraform/r/child", {st_mode=S_IFDIR|0755, st_size=4096, ...}) = 0
```
.. but no calls for the `main.tf` inside - what's going on?

Cheers,
Gavin.

**Comment by mitchellh** [comment]
Fixed

**Comment by bjtucker** [comment]
This hint would be more helpful if it mentioned that local modules need terraform get too. I googled the error and it led me here. I didn't think downloading was what I needed to do for a local file, so I didn't try terraform get.

As an aside, terraform get worked for me but I was surprised by the syntax, minimal as it is. I was expecting to have to tell it what to get. Nope. And my module was all wrong, but terraform get told me what to fix. How cool.
